src/utils.test.js

```diff
@@ -2,13 +2,75 @@ const scrypt = require('scrypt-async');
 const aes = require('aes-js');
 const createHmac = require('create-hmac');
 const crypto = require('crypto');
+const tweetnacl = require('tweetnacl');
+
 const {
   encryptValue,
   decryptValue,
   isInvalidValidPassword,
+  hashDappContent,
 } = require('./utils');
 
+const dAppContent = {
+  name: { 'en-us': 'DApp Name', de: 'DApp Name' },
+  engine: '0.1.0',
+  image: 'iVBORw0KGgoAAAANSUhEUgAAAH0AAAB9CAYAAACPg',
+  used_signing_key:
+    '16a6e448c2e1e209ab559232a98e3d0c25030e05158c22710c207fec1f2e4fbf',
+  code: 'get:function(){return"Text"}',
+};
+
+// hash string that computed from dAppContent
+const hexStringHashDapp =
+  '1678646544417070204e616d65656e2d757344417070204e616d6516a6e448c2e1e209ab559232a98e3d0c25030e05158c22710c207fec1f2e4fbf6765743a66756e6374696f6e28297b72657475726e2254657874227d89504e470d0a1a0a0000000d494844520000007d0000007d08060000008f302e312e30';
```

Comment:
that hash looks very long. `sha3-256` should have 32 bytes length + a few bytes for the multihash.